src/renderers/dom/shared/__tests__/ReactDOMComponent-test.js

```diff
@@ -1024,8 +1024,8 @@ describe('ReactDOMComponent', function() {
           }, '\'"<>&')
         )
       ).toBe(
-        '<div title="&#x27;&quot;&lt;&gt;&amp;" style="text-align:&#x27;&quot;&lt;&gt;&amp;;">' +
-          '&#x27;&quot;&lt;&gt;&amp;' +
+        '<div title="&#39;&quot;&lt;&gt;&amp;" style="text-align:&#39;&quot;&lt;&gt;&amp;;">' +
+          '&#39;&quot;&lt;&gt;&amp;' +
```

Comment:
`escape-html` uses `&#39` instead of `&#x27`, which should be a completely equivalent HTML entity for single quote (and it's one byte less, too).